gst/nnstreamer/tensor_filter/tensor_filter_custom.c

```diff
@@ -37,6 +37,12 @@
 void init_filter_custom (void) __attribute__ ((constructor));
 void fini_filter_custom (void) __attribute__ ((destructor));
 
+static const gchar *custom_accl_support[] = {
+  ACCL_AUTO_STR,
+  ACCL_DEFAULT_STR,
+  NULL
+};
```

Comment:
Q: Is it now mandatory to define `*_accl_support` and `*_checkAvailability` for all sug-plugins?
I think defining this and `*_checkAvailability` for non-accelerated sub-plugins is somewhat trivial and meaningless. 
What happens if we just set `*_checkAvailability` to NULL?

Reply:
Single-shot C-API provides an interface `ml_check_nnfw_availability` which allows users to check if a particular `nnfw` framework is available with a given `hw` backend. `*_checkAvailability` is required to support this interface.

Reply:
Ah, for single-shot APIs. Got it.